src/types/util.ts

```diff
@@ -0,0 +1,9 @@
+export type CamelizeString<T extends PropertyKey> = T extends string
```

Comment:
consider moving that to `./src/util/utils.ts`

Reply:
No